readme.txt

```diff
@@ -9,19 +9,11 @@ License:           GPLv2 or later
 License URI:       https://www.gnu.org/licenses/gpl-2.0.html
 Tags:              performance, images, javascript, site health, measurement, object caching
 
-Performance plugin from the WordPress Performance Team, which is a collection of standalone performance modules.
+Performance plugin from the WordPress Performance Team, which is a collection of standalone performance plugins.
```

Comment:
Please update the plugin description like this everywhere, including in `load.php`, `composer.json` etc.
```suggestion
Performance plugin from the WordPress Performance Team, which is a collection of standalone performance features.
```